[CI] reindex oldEs090Fixture failure (no useful log info)

The task for `oldEs090Fixture` failed (7.6, darwin, see output below) but I can't tell _why_ ant/gradle decided that it failed. The gradle log lacks anything that makes sense to me, and there's nothing reindex related in the google cloud storage bundle.

- https://gradle-enterprise.elastic.co/s/umzh6bdg7lphe/console-log?anchor=1879
- https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.6+multijob-darwin-compatibility/9/console

```
03:13:14 > Task :modules:reindex:oldEs090Fixture
03:13:28 
03:13:28 oldEs090Fixture configuration:
03:13:28 -----------------------------------------
03:13:28   cwd: /private/var/lib/jenkins/workspace/elastic+elasticsearch+7.6+multijob-darwin-compatibility/modules/reindex/build/fixtures/oldEs090Fixture/cwd
03:13:28   command: /var/lib/jenkins/.java/java8/bin/java oldes.OldElasticsearch /private/var/lib/jenkins/workspace/elastic+elasticsearch+7.6+multijob-darwin-compatibility/modules/reindex/build/fixtures/oldEs090Fixture /private/var/lib/jenkins/workspace/elastic+elasticsearch+7.6+multijob-darwin-compatibility/modules/reindex/build/tmp/unzipEs090 true
03:13:28   environment:
03:13:28     JAVA_HOME: /var/lib/jenkins/.java/java8
03:13:28     CLASSPATH: /private/var/lib/jenkins/workspace/elastic+elasticsearch+7.6+multijob-darwin-compatibility/test/fixtures/old-elasticsearch/build/distributions/old-elasticsearch-7.6.0-SNAPSHOT.jar:/private/var/lib/jenkins/.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-core/8.4.0/2746f9ec96f9ce3a345b11f03751136073f7869f/lucene-core-8.4.0.jar
03:13:28 
03:13:28   [run]
03:13:28     "/var/lib/jenkins/.java/java8/bin/java" "$@" > run.log 2>&1 ; if [ $? != 0 ]; then touch run.failed; fi
03:13:28 oldEs090Fixture output:
03:13:28 -----------------------------------------
03:13:28   failure marker exists: false
03:13:28   pid file exists: true
03:13:28   ports file exists: false
03:13:28 
03:13:28   [log]
03:13:28     Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8
03:13:28     Running [/private/var/lib/jenkins/workspace/elastic+elasticsearch+7.6+multijob-darwin-compatibility/modules/reindex/build/tmp/unzipEs090/elasticsearch-0.90.13/bin/elasticsearch, -f, -p, ../pid]
03:13:28     [2020-01-19 19:12:41,868][INFO ][node                     ] [Lightmaster] version[0.90.13], pid[73138], build[249c9c5/2014-03-25T15:27:12Z]
03:13:28     Found pid:  73138
03:13:28     [2020-01-19 19:12:41,869][INFO ][node                     ] [Lightmaster] initializing ...
03:13:28     [2020-01-19 19:12:41,948][INFO ][plugins                  ] [Lightmaster] loaded [], sites []
03:13:28 -----------------------------------------
03:13:30 
03:13:30 > Task :modules:reindex:oldEs090Fixture FAILED
03:14:09 > Task :modules:reindex:oldEs090Fixture#stop
03:14:16 build complete, generating: /private/var/lib/jenkins/workspace/elastic+elasticsearch+7.6+multijob-darwin-compatibility/build/9.tar.bz2
03:19:41 
03:19:41 FAILURE: Build failed with an exception.
03:19:42 
03:19:42 * What went wrong:
03:19:42 Execution failed for task ':modules:reindex:oldEs090Fixture'.
03:19:42 > Failed to start oldEs090Fixture
```

--- a/modules/reindex/build.gradle
+++ b/modules/reindex/build.gradle
@@ -100,7 +100,13 @@ if (Os.isFamily(Os.FAMILY_WINDOWS)) {
 } else {
   /* Set up tasks to unzip and run the old versions of ES before running the
    * integration tests. */
-  for (String version : ['2', '1', '090']) {
+  def versions = ['2', '1', '090']
+  if (Os.isFamily(Os.FAMILY_MAC)) {
+    // 0.90 fails sometimes on mac, given that it is so old, let us disable it
+    // see: https://github.com/elastic/elasticsearch/issues/51202
+    versions = ['2', '1']
+  }
+  for (String version : versions) {
     Task unzip = task("unzipEs${version}", type: Sync) {
       Configuration oldEsDependency = configurations['es' + version]
       dependsOn oldEsDependency

